app/helpers/application_helper.rb

```diff
@@ -7,4 +7,8 @@ def flash_class(level)
         when :alert then "alert alert-error"
     end
   end
+
+  def set_date(date)
```

Comment:
normally we make two methods, render_date and render time.
And set a pattern to run across application.